acmwebsite/controllers/schedule.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+from icalendar import Calendar, Event
+from tg import expose, response
+
+from acmwebsite.lib.base import BaseController
+from acmwebsite.lib.helpers import log
+from acmwebsite.model import DBSession, Meeting
+
+
+class ScheduleController(BaseController):
+    def __init__(self):
+        self.meetings = DBSession.query(Meeting).order_by(Meeting.date).all()
+
+    @expose('acmwebsite.templates.schedule')
+    @expose('json', exclude_names=['page'])
+    def _index(self):
+        """Handle the schedule page."""
+
+        # Allow CORS
+        response.headers.add("Access-Control-Allow-Origin", "*")
+        return dict(page='schedule', meetings=self.meetings.filter(
+            Meeting.date > datetime.datetime.now() - datetime.timedelta(hours=3)
+        ))
+
+    @expose(content_type='text/calendar')
+    def acm(self):
```

Comment:
Any way to make the url `/schedule.ics` instead?

Reply:
So it will end up being `https://acm.mines.edu/schedule.ics`?

Reply:
Yes, that is if the site URL is `https://acm.mines.edu`.